code/modules/mob/living/carbon/human/species/outsider/event.dm

```diff
@@ -0,0 +1,215 @@
+	knockout_message				The message when someone gets a lucky shot on the head.
+	cloning_modifier				The modifier given when they are by chance cloned.
+
+	Any 'heat' or 'pressure' vars.	Environmental survivability.
+
+	speech_bubble_appearance		Part of icon_state to use for speech bubbles when talking.	See talk.dmi for available icons.
+
+	slowdown						Passive movement speed malus (or boost, if negative)
+	move_trail						What effect marks are left when walking
+	has_floating_eyes				Whether the eyes can be shown above other icons
+	has_glowing_eyes				Whether the eyes are shown above all lighting
+	water_movement					How much faster or slower the species is in water
+	snow_movement					How much faster or slower the species is on snow
+	item_slowdown_mod				How affected by item slowdown the species is. Multiplier.
+
+ */
+
+/datum/species/event1 //Essentially, by default a 'better' human.
+	name = SPECIES_EVENT1
+	name_plural = SPECIES_EVENT1
+	primitive_form = SPECIES_MONKEY
+	unarmed_types = list(/datum/unarmed_attack/stomp/event1, /datum/unarmed_attack/kick/event1, /datum/unarmed_attack/punch/event1, /datum/unarmed_attack/bite/event1)
+	blurb = "We're not quite sure where these things came from. Are you?"
+	num_alternate_languages = 3
+	species_language = LANGUAGE_GALCOM
+	secondary_langs = list()
+	name_language = null // Use the first-name last-name generator rather than a language scrambler
+
+	min_age = 0
+	max_age = 999
+
+	health_hud_intensity = 1.5
+
+	flags = NO_MINOR_CUT | NO_HALLUCINATION | NO_INFECT
+
+	vision_flags = SEE_SELF
+	darksight = 7
+
+	brute_mod = 0.9
+	burn_mod = 0.9
+	oxy_mod = 0.9
+	toxins_mod = 0.9
+	radiation_mod = 0.9
+	flash_mod = 0.9
+	sound_mod = 0.9
+	chemOD_mod = 0.9
+	siemens_coefficient = 0.9
+
+	spawn_flags = SPECIES_IS_RESTRICTED
+	appearance_flags = HAS_SKIN_TONE | HAS_EYE_COLOR
+
+	//Far more organs, due to the issues with adding them and their respective verbs mid round, I'll include them here.
+	//Colormatch organs will change to match the blood color of the species.
+	has_organ = list(
+		O_HEART =    /obj/item/organ/internal/heart/grey/colormatch,
+		O_LUNGS =    /obj/item/organ/internal/lungs/grey/colormatch,
+		O_LIVER =    /obj/item/organ/internal/liver/grey/colormatch,
+		O_KIDNEYS =  /obj/item/organ/internal/kidneys/grey/colormatch,
+		O_BRAIN =    /obj/item/organ/internal/brain/grey/colormatch,
+		O_EYES =     /obj/item/organ/internal/eyes/grey/colormatch,
+		O_PLASMA =   /obj/item/organ/internal/xenos/plasmavessel/grey/colormatch,
+		O_ACID =     /obj/item/organ/internal/xenos/acidgland/grey/colormatch,
+		O_HIVE =     /obj/item/organ/internal/xenos/hivenode/grey/colormatch,
+		O_RESIN =    /obj/item/organ/internal/xenos/resinspinner/grey/colormatch
+		)
+
+	var/use_bodyshape = SPECIES_HUMAN
+	var/waterbreathing = 0
+	var/overcome_gravity = 0
+	var/hover = 0
+
+/datum/species/event1/proc/set_limbset(var/setnum = 1) //Will require existing ones to be respawned for changes to take effect.
```

Comment:
Since it's intended for adminbus, you could add another proc that `input()`'s the user to get the constants `set_limbset()` uses from human-readable (i.e., actual words) options